op-acceptance-tests/tests/interop/reorgs/init_exec_msg_test.go

```diff
@@ -233,36 +202,11 @@ func TestReorgInitExecMsg(gt *testing.T) {
 	}
 
 	// continue sequencing with op-node
-	{
-		newUnsafeHeadRef := sys.L2ChainA.UnsafeHeadRef()
-		l.Info("Continue sequencing with consensus node (op-node)", "unsafeHead", newUnsafeHeadRef)
-
-		err := sys.L2CLA.Escape().RollupAPI().StartSequencer(ctx, newUnsafeHeadRef.Hash)
-		require.NoError(t, err, "Expected to be able to start sequencer on rollup node")
-
-		// wait for the sequencer to become active
-		var active bool
-		err = wait.For(ctx, 1*time.Second, func() (bool, error) {
-			active, err = sys.L2CLA.Escape().RollupAPI().SequencerActive(ctx)
-			return active, err
-		})
-		require.NoError(t, err, "Expected to be able to call SequencerActive API, and wait for an active state for sequencer, but got error")
-
-		l.Info("Rollup node sequencer", "active", active)
-	}
+	sys.L2CLA.StartSequencer()
```

Comment:
question: why are we calling the sequencer API to stop L2CL, but not stopping using `sys.L2CLA.Stop()`?